stac_mjx/main.py

```diff
@@ -75,14 +75,13 @@ def run_stac(
     if cfg.stac.skip_transform == 1:
         logging.info("skipping transform()")
         return fit_path, None
-    elif kp_data.shape[0] % cfg.model.N_FRAMES_PER_CLIP != 0:
-        raise ValueError(
-            f"N_FRAMES_PER_CLIP ({cfg.model.N_FRAMES_PER_CLIP}) must divide evenly with the total number of mocap frames({kp_data.shape[0]})"
-        )
+    # elif kp_data.shape[0] % cfg.model.N_FRAMES_PER_CLIP != 0:
+    #     raise ValueError(
+    #         f"N_FRAMES_PER_CLIP ({cfg.model.N_FRAMES_PER_CLIP}) must divide evenly with the total number of mocap frames({kp_data.shape[0]})"
+    #     )
```

Comment:
Why did you need to take this out?

Reply:
Since my data is not the same length for every trial it throws and error so I needed to comment out until charles handles the clipping